### PR TITLE
Update for Julia 1.0

### PR DESCRIPTION
**Pull request by Ellipse0934**
Fix compatibility issues.

### CODE REVIEW

**Comment by simonbyrne** [comment]
Thanks. I don't have my pi set up, so I take it that this works and tests pass?

**Comment by Ellipse0934** [comment]
Yes, they do. I tested everything and it works.
Maybe, change to version to 0.2.0 ?

**Comment by Ellipse0934** [comment]
Actually, scratch that. The pressure is being reported as -6 millibar which is clearly wrong. That one test fails and need to change `using Base.Test` to `using Test` in the test files. I have some work right now, I will fix it in a couple of hours and report when done.

**Comment by simonbyrne** [comment]
No worries, thanks for looking into it. I'll try to test it this weekend as well.

**Comment by Ellipse0934** [comment]
Issues have been fixed. Although my pressure sensor is showing 936 millibar which is abnormally low. Since python's sense hat library is showing the same the code is correct although the test will fail.

File: REQUIRE [hunk 1]
```diff
@@ -1,3 +0,0 @@
-julia 0.6
-ColorTypes
-FixedPointNumbers
```
Comment (simonbyrne) [inline REQUIRE]: You need to keep REQUIRE around for attobot (see https://github.com/attobot/attobot/issues/50).

File: src/ioctl.jl [hunk 2]
```diff
@@ -25,7 +25,7 @@ function ioctl(fd::Cint, request::Integer, arg::Integer)
     return ret
 end
 function ioctl(fd::Cint, request::Integer, arg)
-    ret = ccall(:ioctl, Cint, (Cint, Culong, Ptr{Void}...), fd, request, arg)
+    ret = ccall(:ioctl, Cint, (Cint, Culong, Ptr{Nothing}...), fd, request, arg)
```
Comment (simonbyrne) [inline src/ioctl.jl]: Use `Cvoid` instead of `Nothing` (yes I know it's an alias, but better to be clearer).

**Comment by simonbyrne** [comment]
Feel free to increase the tolerance of the test: they're really to just check we're getting the correct order of magnitude.

**Comment by simonbyrne** [comment]
Thanks!

**Pull request merged**